aries_cloudagent/revocation/models/issuer_rev_reg_record.py

```diff
@@ -138,8 +138,7 @@ def _check_url(self, url) -> None:
         if self.state != IssuerRevRegRecord.STATE_INIT:
             raise RevocationError(
                 "Revocation registry {} in state {}: cannot generate".format(
-                    self.revoc_reg_id,
-                    self.state
+                    self.revoc_reg_id, self.state
```

Comment:
Is there any way to shut flake up about this kind of thing? I can read left-to-right, or I can read up-to-down. When you make me read both, it's a wall of text: poor choice when details matter.

Reply:
It's the 'black' formatter, as in you can get any colour you want as long as it's black. :|